UPGRADE-6.3.md

```diff
@@ -103,3 +103,80 @@ None.
 ## Database Fields
 
 None.
+
+## Backend Theme Library
+
+The Backend was upgraded to Bootstrap 4.1.3.
+
+See the [Bootstrap Migration Guide](https://getbootstrap.com/docs/4.1/migration/) for needed changes to your backend modules.
+
+During the upgrade to Bootstrap 4 the following styles where checked and these are our findings:
+
+btn-default -> btn-secondary
+- Some TCMSField types and TCMSTableEditors so check yours) 
```

Comment:
) too much

What does this line mean? There is html code in those classes?

Reply:
Yes. It is. In most of older/all fieldtypes there is HTML code right in the class.